src/Persistence.RavenDB/RavenDBViewModelHelper.cs

```diff
@@ -0,0 +1,20 @@
+﻿// <copyright file="RavenDBViewModelHelper.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.ViewModels.Persistence.RavenDB
+{
+    /// <summary>
+    /// Helper class used for code shared between <see cref="RavenDBViewModelReader"/> and <see cref="RavenDBViewModelWriter"/>.
+    /// </summary>
+    internal static class RavenDbViewModelHelper
```

Comment:
Comments seems to have disappeared on this one. 2 Letter acronyms should be left categorised based on MS Guidelines https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)